symengine/cwrapper.h

```diff
@@ -177,7 +177,7 @@ unsigned long integer_get_ui(const basic s);
 CWRAPPER_OUTPUT_TYPE integer_get_mpz(mpz_t a, const basic s);
 
 //! Assign to s, a rational i/j. Returns 0 if either i or j is not an integer.
-CWRAPPER_OUTPUT_TYPE rational_set(basic s, const basic i, const basic j);
+int rational_set(basic s, const basic i, const basic j);
```

Comment:
instead of `0` and `1`, you can return an error code here and keep the return type `CWRAPPER_OUTPUT_TYPE`